.github/workflows/master.yml

```diff
@@ -72,16 +72,18 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-io-
       - name: Build with Maven
-        if: ${{ ! contains(matrix.profiles, 'wildcardSuites') }}
+        if: ${{ ! contains(matrix.profiles, 'tpcds.tests=true') }}
         run: mvn clean install ${{ matrix.profiles }} -Dmaven.javadoc.skip=true -V
       - name: Wild Card Suites Tests
-        if: ${{ contains(matrix.profiles, 'wildcardSuites') }}
-        run: mvn clean install -DskipTests -pl :kyuubi-spark-sql-engine,:kyuubi-common,:kyuubi-ha,:kyuubi-zookeeper && mvn -Dtest=none ${{ matrix.profiles }} test
+        if: ${{ contains(matrix.profiles, 'tpcds.tests=true') }}
+        run: |
+          mvn clean install ${{ matrix.profiles }} -DskipTests -pl :kyuubi-spark-sql-engine,:kyuubi-common,:kyuubi-ha,:kyuubi-zookeeper
+          mvn test ${{ matrix.profiles }} -Dtest=none -DwildcardSuites=org.apache.kyuubi.operation.tpcds.TPCDSOutputSchemaSuite,org.apache.kyuubi.operation.tpcds.TPCDSDDLSuite
       - name: Code coverage
         if: ${{ matrix.profiles == '' }}
         run: bash <(curl -s https://codecov.io/bash)
       - name: Detected Dependency List Change
-        if: ${{ ! contains(matrix.profiles, 'spark-3.2-snapshot') && ! contains(matrix.profiles, 'wildcardSuites') }}
+        if: ${{ ! contains(matrix.profiles, 'spark-3.2-snapshot') && ! contains(matrix.profiles, 'tpcds.tests=true') }}
```

Comment:
instead of using matrix I prefer to move tpcds tests to another separate job like `rat`

Reply:
And not related to this PR, we can mv `spark-3.2-snapshot` to a another `yml` with nightly build enabled?